app/src/main/java/com/example/trashminder/utils/TrashType.kt

```diff
@@ -0,0 +1,33 @@
+package com.example.trashminder.utils
+
+import android.content.Context
+import com.example.trashminder.R
+
+enum class TrashType {
+    Plastic,
+    Metals,
+    Paper,
+    Glass,
+    Residuals
+}
+
+fun TrashType.toLocalizedString(context: Context): String {
```

Comment:
This can be renamed `toResourceId():Int` and make it return just the resource id